chef/cookbooks/crowbar-pacemaker/recipes/drbd.rb

```diff
@@ -25,15 +25,24 @@
 claimed_disks = BarclampLibrary::Barclamp::Inventory::Disk.claimed(node, claim_string).sort
 
 if claimed_disks.empty? and not unclaimed_disks.empty?
+
+  disk_name_to_claim = node["crowbar_wall"].disks_roles.find { |_, role| role == claim_string }[0] rescue ""
```

Comment:
That's the part I don't understand: why not directly claim the disk from barclamp-crowbar? That what really what I meant with https://github.com/crowbar/barclamp-crowbar/pull/1306/files#r34164794 :-)

This would mean no change in the other barclamps doing disk claiming.